backstop/tests/table.html

```diff
@@ -3798,5 +3849,16 @@ <h2>Empty state</h2>
         });
       });
     </script>
+
+    <script>
+      var elements = document.getElementsByClassName('iui-dropdown');
```

Comment:
Please don't use `var` anywhere ever for any reason at all.
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
I dont see you use this variable anywhere too

Reply:
use of `var` has been removed